torch/distributed/algorithms/model_averaging/averagers.py

```diff
@@ -109,5 +113,5 @@ def average_parameters(self, params):
             self.step >= self.warmup_steps
             and (self.step - self.warmup_steps) % self.period == 0
         ):
-            utils.average_parameters(iter(params), self.process_group)
-        self.step += 1
+            utils.average_parameters(params, self.process_group)
+        self.step += 1
```

Comment:
https://github.com/pytorch/pytorch/pull/74894#issuecomment-1082605348
Add a newline at the end of this file to fix your linter error.